src/browser/components/UpdaterPage/UpdaterPage.stories.jsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {action} from '@storybook/addon-actions';
+
+import UpdaterPage from '../UpdaterPage.jsx';
+import '../../css/components/UpdaterPage.css';
+
+/*
+appName: propTypes.string.isRequired,
+notifyOnly: propTypes.bool.isRequired,
+isDownloading: propTypes.bool.isRequired,
+progress: propTypes.number,
+onClickInstall: propTypes.func.isRequired,
+onClickDownload: propTypes.func.isRequired,
+onClickReleaseNotes: propTypes.func.isRequired,
+onClickRemind: propTypes.func.isRequired,
+onClickSkip: propTypes.func.isRequired,
+*/
+const appName = 'Storybook App';
```

Comment:
What's this about?

Reply:
Just for [storybook](https://storybook.js.org/) previewing. It appears as application name as a part of dialog sentences. To white labeling the app (e.g. uchat), the app name is injected to the component as a prop.

You can see this name at http://localhost:9001 while running `npm run storybook`.